morphir/runtime/test/jvm/src/org/finos/morphir/runtime/EvaluatorMDMTests.scala

```diff
@@ -1199,6 +1199,20 @@ object EvaluatorMDMTests extends MorphirBaseSpec {
         testEvalMultiple("diffInYears")("localDateTests", "diffInYearsTest", List(localDate, localDate.plusYears(2)))(
           Data.Int(2)
         ),
+        testEvalMultiple("diffInYearsAdjacent")(
```

Comment:
For consistency, can you update the elm test examples with these test cases? Also add them for the LocalTime ones you added.

Reply:
fixed